examples/mnist_hierarchical_rnn.py

```diff
@@ -0,0 +1,70 @@
+"""
+An example of hierarchical RNN to predict MNIST digits
+Hierarchical model is implemented elegantly with the
```

Comment:
No need to qualify the code as "elegant". The docstring needs more detailed explanations of what the model does, how, and why it is interesting (what you can apply it to, etc).